### ISSUE TITLE
'q' is null or not an object in Internet Explorer

### ISSUE THREAD

**Issue by jbarger** [dotmaster/Touchable-jQuery-Plugin#1]
Getting errors under IE. "Object doesn't support this property or method." Line 55 char 5 when calling touchable against an element.

**Comment by dotmaster** [comment]
hi, thank you for your bug report, am busy right now, but will check that at the weekend

**Comment by Protopop** [comment]
Thank you for the great plugin.

How would I get the current mouseX / fingerX position from inside the 'touchablemove'?

On the desktop  i can use e.pageX:
       $(document).mousemove(function(e){
           $mousePosX = e.pageX;...

but from inside the bound event e.pageX doesn't return the mouse/fingerX

```
   $t4.bind('touchablemove', function(e, touch){
       $mousePosX =  //how could i get the current mouseX (or fingerX) from here?...
```

I am making a mouse / touch compatible Drag. If I can get the X position i will use it to set the Element position under the mouse/finger.

**Comment by dotmaster** [comment]
hi there 
notice, that each event gets passed a touch object, besides the normal event object. The Touch object has the following properties:

startTouch: this is where the touch or mousedown event originated
currentTouch: this is where we are right now with our finger or mouse <--this is your case
previousTouch: for internal calculations of the previous position of the mouse pointer or finger, used for deltas see below
currentDelta: measured from previous move event
currentStartDelta, currentPosition: the relative position, measured from start calculated in different ways, but should be the same

**Comment by Protopop** [comment]
Thank you:) I got the drag to work with your help.
I use "touch.currentTouch.x" to get the current mouse/finger X position

```
    $t4.bind('touchablemove', function(e, touch){
        $mouseDownX = touch.startTouch.x;
        $mousePosition = touch.currentTouch.x....
```

works great on iPad and desktop. Im really glad you created this plugin.

**Comment by dotmaster** [comment]
great to hear ;)

**Comment by dotmaster** [comment]
should be fixed with version 0.0.2
tested IE 7/8
